README.md

```diff
@@ -59,6 +59,301 @@ drwxr-xr-x  2 you  staff    68 Mar 29 14:16 .cache
 ```
 *Security notice:* the file `.netrc`, created or updated by `conjur authn login`, contains a user identity credential that can be used to access the Conjur API. You should remove it after use or otherwise secure it like you would another netrc file.
 
+## Usage
+
+```
+NAME
+    conjur - Command-line toolkit for managing roles, resources and privileges
+
+SYNOPSIS
+    conjur [global options] command [command options] [arguments...]
+
+GLOBAL OPTIONS
+    --help    - Show this message
+    --version - Display the program version
+```
+
+### Commands
+
+[authn](#authn)       - Login and logout
+
+[check](#check)       - Check for a privilege on a resource
+
+[env](#env)         - Use values of Conjur variables in local context
+
+[help](#help)        - Shows a list of commands or help for one command
+
+[host](#host)        - Manage hosts
+
+[hostfactory](#hostfactory) - Manage host factories
+
+[init](#init)        - Initialize the Conjur configuration
+
+[ldap-sync](#ldap-sync)   - LDAP sync management commands
+
+[list](#list)        - List objects
+
+[plugin](#plugin)      - Manage plugins
+
+[policy](#policy)      - Manage policies
+
+[pubkeys](#pubkeys)     - Public keys service operations
+
+[resource](#resource)    - Manage resources
+
+[role](#role)        - Manage roles
+
+[show](#show)        - Show an object
+
+[user](#user)        - Manage users
+
+[variable](#variable)    - Manage variables
+
+#### authn
```

Comment:
Given that this is a long list and the reader might not know what section they are in, you should add the `conjur ` prefix to these and also enclose them in backticks. Eg:
```
`conjur authn`
```

Reply:
I'm not a big fan of the header tags and the backticks at the same time, but I added it